app/assets/stylesheets/header.css.scss

```diff
@@ -0,0 +1,34 @@
+@import "bootstrap";
+
+.navbar-brand { position: relative; z-index: 2; }
```

Comment:
Agreed. Seems the majority of front-enders are against one-liners. Can you just make all of them multi-line? Thanks.